pkg/capabilities/triggers/mercury_trigger.go

```diff
@@ -27,7 +27,7 @@ const defaultTickerResolutionMs = 1000
 // TODO pending capabilities configuration implementation - this should be configurable with a sensible default
 const defaultSendChannelBufferSize = 1000
 
-type config struct {
+type Config struct {
 	// strings should be hex-encoded 32-byte values, prefixed with "0x", all lowercase, minimum 1 item
 	FeedIDs []string `json:"feedIds" jsonschema:"pattern=^0x[0-9a-f]{64}$,minItems=1"`
 	// must be greater than 0
```

Comment:
Shouldn't this be replaced with the generated structs? This tests if what is generated matches what is needed.

Reply:
I'll verify they are the same, then do the replacement :D.  If they aren't, I'll update them to be